docs/04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md

```diff
@@ -2,19 +2,14 @@
 title: Enable Kyma with Runtime Agent
 ---
 
-To enable Kyma with Runtime Agent, follow the [cluster Kyma installation with specific components](02-install-kyma.md#install-specific-components) and add the `compass-runtime-agent` module to the list of components. The default [legacy mode](../../05-technical-reference/00-architecture/ac-01-application-connector-components.md#application-operator) used in Kyma does not support integration with Compass. For that reason, before you start the installation, apply the following ConfigMap which disables components used in the legacy mode, such as Application Registry and Connector Service:
+[Runtime Agent](../../01-overview/main-areas/application-connectivity/ra-01-runtime-agent-overview.md) is a component that is used in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md) of Application Connectivity. 
+By default, Kyma uses the [legacy Application Connectivity mode](../../01-overview/main-areas/application-connectivity/README.md), which does not support integration with Compass.
```

Comment:
I wouldn't repeat the link from the previous sentence here. Alternatively, you can add such a sentence:
```Read the general document about Application Connectivity to learn more about the [legacy and Compass modes](link).```

... or sth like that.

Reply:
I'm not 100% sold on that TBH... 
The document should be as concise as possible, so IMO it's better to duplicate a link -- which, by the way, nicely highlights the two concepts -- than add a whole new NOTE/sentence that doesn't really add new information.
Let's discuss quickly.